components/Form/Form.js

```diff
@@ -0,0 +1,185 @@
+import { useState } from "react";
+import {
+  Form,
+  Fieldset,
+  Legend,
+  Label,
+  Input,
+  Textarea,
+  Select,
+  Checkbox,
+  FormButton,
+  SomeoneElse,
+} from "./Form.styled";
+import PlanList from "../PlanList/PlanList";
+export default function Plan() {
+  const [plan, setPlan] = useState([]);
+  const [forSomeoneElse, setForSomeoneElse] = useState(false);
+  const handleForChange = (event) => {
+    setForSomeoneElse(event.target.value === "Someone else");
+  };
+
+  const handleSubmit = (event) => {
+    event.preventDefault();
+    const formData = new FormData(event.target);
+    const frequency = [];
+    for (let [key, value] of formData.entries()) {
+      if (key.startsWith("frequency")) {
+        frequency.push(value);
+      }
+    }
+    const plan = {
+      ...Object.fromEntries(formData.entries()),
+      frequency,
+    };
+    setPlan((prevPlan) => [...prevPlan, plan]);
+  };
+
+  console.log(plan);
+
+  return (
+    <>
+      <Form onSubmit={handleSubmit}>
+        <Fieldset>
+          <Legend> Medications plan</Legend>
+          <label htmlFor="for">
+            {" "}
+            For :
+            <Checkbox
+              type="radio"
+              name="for"
+              value="Me"
+              onChange={handleForChange}
+            />{" "}
+            <span>Me</span>{" "}
+          </label>
+          <Checkbox
+            type="radio"
+            name="for"
+            defaultValue="Someone else"
+            onChange={handleForChange}
+          />{" "}
+          <span>Someone else</span>
+          <br />
+          {forSomeoneElse && (
+            <SomeoneElse>
+              <Label htmlFor="name">
+                {" "}
+                Name :
+                <Input type="text" name="name" />
+              </Label>
+              <Label htmlFor="age">
+                {" "}
+                Age :
+                <Input type="number" name="age" />
+              </Label>
+              <Label htmlFor="gender">
+                {" "}
+                Gender :
+                <Select name="gender">
+                  <option defaultValue="Male">Male</option>
```

Comment:
`defaultValue` is not a prop of the `option` tag as far as I know. You should set a `value` prop for each option. Then you control the default value by setting the `value` prop of the `select` to the desired value. [How to create a React Select](https://www.robinwieruch.de/react-select/)

Reply:
I have already change "defaultValue" to "value"